pvlib/tracking.py

```diff
@@ -545,18 +544,17 @@ def singleaxis(apparent_zenith, apparent_azimuth,
     surface_azimuth[surface_azimuth >= 360] -= 360
 
     # Calculate surface_tilt
-    # Use pandas to calculate the sum because it handles nan values better.
```

Comment:
The issue referenced in the comment above is that in earlier versions of pandas the sum of a Series (or along one axis of a DataFrame) of all NaNs returned NaN (so long as bottleneck was not installed). That behavior changed a year or two ago, and now ``pd.Series([NaN, NaN]).sum() = 0``. So there is no longer a reason to prefer pandas. I think my original motivation was to preserve input nan values, and that no longer happens.

Reply:
new ``test_nans`` ensures that input nans are properly handled.